### DISCUSSION TITLE
How combine bar and line in the same graph ?

### DISCUSSION THREAD

**Discussion by Daniel-Mompean** [grafana/grafana#40532] [Dashboards, Panels & Visualizations]
I can't combine bar and line in the same graph, this is a very used feature !

**Comment by ryantxu** [comment]
If your x axis is time, the "timeseries" panel does this fine -- but when it is not, no good options.  Considering this in:
https://github.com/grafana/grafana/pull/43302

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
